packages/client/modules/userDashboard/components/OrgBilling/OrgBillingDangerZone.tsx

```diff
@@ -54,15 +58,23 @@ const OrgBillingDangerZone = (props: Props) => {
         ...ArchiveOrganization_organization
         isBillingLeader
         tier
+        viewerOrganizationUser {
```

Comment:
-1 I see that if the user has the feature flag, then this component is used in a different component tree. I think this component should not know how it's used. Instead I see 2 possible cleaner solutions:
* add a prop `isWide` to the component. hacky but good enough for now
* add a style/className property and set the style directly on the parent